docs/glossary/glossary.md

```diff
@@ -1,5 +1,11 @@
 # Splinter Glossary
 
+## Biome
+
+Biome is a module in libsplinter containing several sub modules that provide
```

Comment:
This use of "sub modules" should be consistent with the choice made do to the previous comment.